README.md

```diff
@@ -43,9 +43,16 @@ You can install Akira by compiling it from the source, here's the list of depend
  - `gobject-2.0`
  - `libxml-2.0`
  - `gtksourceview-3.0`
+ - `libjson-glib-1.0`
  - `goocanvas-2.0`
+ - `libarchive`
  - `cairo`
  - `meson`
+ 
+ In Elementary OS 0.3 Freya and later, resolve dependencies by typing:
+ ```
+ sudo apt-get install -y valac meson build-essential libgtk-3-dev libgranite-dev libxml++2.6-dev libgoocanvas-2.0-dev libarchive-dev glib-2.0 libgtksourceview-3.0-dev libjson-glib-dev
```

Comment:
Can you please remove this part? The readme shouldn't ask anyone to execute anything as a sudoer.

Reply:
Ok